UBCMint/MINT-Open-Source
# tammykim00723/neu 47 database for headset recordings

**Here give examples of the changes you've made in this pull request. Include an itemized list if you can.**

- Created a database called recordings, which has headset, name, time as its attributes
- Created a get request API that lets you get all the data information from the recordings database

**How to test?**
Create GET request in Postman with the API endpoint http://localhost:3000/server/api/recordings/getHeadsetRecordings

<img width="1084" alt="Screen Shot 2024-03-30 at 11 10 52 PM" src="https://github.com/UBCMint/MINT-Open-Source/assets/65116042/1827130f-f7d1-438e-ac79-c925ac41fb4c">


On https://local.drizzle.studio, should display an empty table like this:
<img width="521" alt="Screen Shot 2024-03-30 at 11 15 10 PM" src="https://github.com/UBCMint/MINT-Open-Source/assets/65116042/c98646c0-283d-4dea-b9bd-a776e3d3f89d">
